au/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 Aurora Operations, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_path(GET CMAKE_CURRENT_SOURCE_DIR PARENT_PATH AU_BASE_DIRECTORY)
+
+#
+# Private implementation detail targets
+#
+
+add_library(stdx INTERFACE)
```

Comment:
Should this be moved into au/stdx/CMakeLists.txt? Then, au/CMakeLists would have `add_subdirectory(stdx)`.

Reply:
I wanted to keep our CMake support analogous to our bazel support.  In this case, we use the `stdx` and `utility` subfolders purely for organization.